pkg/util/iptables/iptables.go

```diff
@@ -481,7 +481,7 @@ func trimhex(s string) string {
 // of hack and half-measures.  We should nix this ASAP.
 func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...string) (bool, error) {
 	iptablesSaveCmd := iptablesSaveCommand(runner.protocol)
-	klog.V(1).Infof("running %s -t %s", iptablesSaveCmd, string(table))
+	klog.V(1).InfoS("running iptables command", "iptablesSaveCmd", iptablesSaveCmd, "table", string(table))
```

Comment:
```suggestion
	klog.V(1).InfoS("Running iptables command", "command", iptablesSaveCmd, "IPtable", string(table))
```

Reply:
Here I'm not sure about `table` vs `IPtable`. Technically table is unique enough name that I don't expect it to be used in any other context then IP table. Assuming that this key will mostly be used within networking code then there is nothing against just using simpler `table`.

Reply:
"table" is unambiguous in this context. I assume we are not trying to have keys that are guaranteed to be unique across all of kubernetes